features/git-sync/all_branches/feature_branch_merge_main_branch_conflict/without_remote_origin/first_branch/without_open_changes.feature

```diff
@@ -0,0 +1,108 @@
+Feature: git sync --all: handling merge conflicts between feature branch and main branch (without open changes or remote repo)
```

Comment:
Please apply the changes outlined above to all the feature specs in this PR. Thanks!  :)